bin/node/node/src/service.rs

```diff
@@ -16,13 +16,14 @@
 
 //! Service and ServiceFactory implementation. Specialized wrapper over substrate service.
 
-use bridge_node_runtime::{self, opaque::Block, RuntimeApi};
-use grandpa::{self, FinalityProofProvider as GrandpaFinalityProofProvider, SharedVoterState, StorageAndProofProvider};
-use sc_client_api::ExecutorProvider;
-use sc_consensus::LongestChain;
+use bridge_node_runtime::{self, Block, RuntimeApi};
```

Comment:
Is it necessary for us to have that file at all? Isn't it possible to reference the service builder from substrate somehow?

Reply:
No, I don't think so